src/components/ProblemsPage/SearchBox.tsx

```diff
@@ -36,7 +36,7 @@ export default function SearchBox(props: UseSearchBoxProps): JSX.Element {
       </div>
       <input
         id="search"
-        className="block w-full pl-12 pr-3 py-3 rounded-md bg-blue-100 dark:bg-blue-800 placeholder-gray-900 dark:placeholder-blue-300 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue transition text-black dark:text-white"
+        className="block w-full pl-12 pr-3 py-3 rounded-md bg-blue-100 dark:bg-blue-100 placeholder-gray-900 dark:placeholder-blue-900 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue transition text-black dark:text-white"
```

Comment:
nice! can you also change the text color in dark mode? also, what are your thoughts on making the background a bit darker in dark mode, maybe `bg-blue-200`?

Thanks for making this change!

Reply:
Which text specifically? And to what color? Also for the background color is it the one inside the search box or outside?

Reply:
which text specifically and what color? Also should I change the outside background or searchbox background. Thanks!

Reply:
the text inside the search box. Up to you which color -- it's just white on light background right now which is not good.

<img width="870" alt="image" src="https://github.com/cpinitiative/usaco-guide/assets/7865976/587ce9b8-a5d3-4056-b4de-71586a7573b0">

I was referring to the background of the input box.

Reply:
i made another pull request